v1.1/monitor-cockroachdb-with-prometheus.md

```diff
@@ -150,12 +169,14 @@ If you like, you can connect [Alertmanager](https://prometheus.io/docs/alerting/
 
 6. Start the Alertmanager server, with the `--config.file` flag pointing to the configuration file:
 
+    {% include copy-clipboard.html %}
     ~~~ shell
     $ alertmanager --config.file=simple.yml
     ~~~
 
 7. In the shell running Prometheus, use **CTRL + C** to stop Prometheus and then restart it with the `--config.file` flag pointing to the Prometheus configuration file and the `--alertmanager.url` flag pointing to the machine running Alertmanager:
 
+    {% include copy-clipboard.html %}
     ~~~ shell
     $ prometheus --config.file=prometheus.yml \
```

Comment:
The alertmanager is no longer specified on the commandline, but in the config file. See https://github.com/cockroachdb/cockroach/pull/24253/files#diff-c77c467946df9e8c30e08a81f6d8bebf for the new section.